src/app/modules/shared/models/activity.model.ts

```diff
@@ -1,5 +1,10 @@
 export interface Activity {
-    id: string;
-    name: string;
-    description: string;
+  id: string;
+  created_at?: string;
+  updated_at?: string;
+  created_by?: string;
+  updated_by?: string;
```

Comment:
We are not going to use these fields in the UI so I think we should not include them.